tools/profiling/memory/memory_diff.py

```diff
@@ -47,36 +47,42 @@
         (rb'client call memory usage: ([0-9\.]+) bytes per call', float),
     'server call':
         (rb'server call memory usage: ([0-9\.]+) bytes per call', float),
+    'client channel':
+        (rb'client channel memory usage: ([0-9\.]+) bytes per channel', float),
+    'server channel':
+        (rb'server channel memory usage: ([0-9\.]+) bytes per channel', float),
 }
 
 _SCENARIOS = {
     'default': [],
     'minstack': ['--scenario_config=minstack'],
 }
 
-
+_BENCHMARKS = {
+    'call': ['--benchmark_names=call','--size=50000'],
+    'channel': ['--benchmark_names=channel','--size=20000'],
```

Comment:
meant to be 10000, not 20000. Will update with a commit once tests finish running

Reply:
Updated it!